compute/cloud-client/src/main/java/compute/disks/AttachDisk.java

```diff
@@ -0,0 +1,97 @@
+/*
+ * Copyright 2023 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package compute.disks;
+
+// [START compute_regional_disk_attach]
+// [START compute_disk_attach]
+
+import com.google.cloud.compute.v1.AttachDiskInstanceRequest;
+import com.google.cloud.compute.v1.AttachedDisk;
+import com.google.cloud.compute.v1.InstancesClient;
+import com.google.cloud.compute.v1.Operation;
+import java.io.IOException;
+import java.util.concurrent.ExecutionException;
+import java.util.concurrent.TimeUnit;
+import java.util.concurrent.TimeoutException;
+
+public class AttachDisk {
+
+  public static void main(String[] args)
+      throws IOException, ExecutionException, InterruptedException, TimeoutException {
+    // TODO(developer): Replace these variables before running the sample.
+    // Project ID or project number of the Cloud project you want to use.
+    String projectId = "your-project-id";
+
+    // Name of the zone in which the instance you want to use resides.
+    String zone = "zone-name";
+
+    // Name of the compute instance you want to attach a disk to.
+    String instanceName = "instance-name";
+
+    // Full or partial URL to a persistent disk that you want to attach. This can be either
```

Comment:
Full or partial URL for a persistent disk...
'for' or 'of' a persistent disk sounds better. WDYT?

Reply:
Modifying it to "of". Thank you.